src/components/statistics.jsx

```diff
@@ -0,0 +1,28 @@
+export function Statistics() {
```

Comment:
```suggestion
export function StatisticsPage() {
```

just matching the convention that me and Angela used